linode/databasepostgresqlv2/resource_test.go

```diff
@@ -319,6 +320,11 @@ func TestAccResource_fork(t *testing.T) {
 					resource.TestCheckResourceAttr(resNameSource, "pending_updates.#", "0"),
 				),
 			},
+			{
+				ResourceName:      resNameSource,
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
```suggestion
				ImportStateVerify: true,
				ImportStateVerifyIgnore: []string{"updated"},
```
Probably best to ignore `updated` here too just to be safe. Also, can you add the same thing in the `TestAccResource_complex` MySQL test? I was seeing some intermittent failures with this test yesterday that were caused by the `updated` field not quite matching up.

Reply:
Updated!